### PR TITLE
Add information notification pattern & docs

### LINKED ISSUE #1060: Update notification styles according to design spec
Design spec: https://github.com/ubuntudesign/vanilla-design/tree/master/Notifications

Notes:
- Add information style
- Information uses information colour
- Information has no icon, for now

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Add information notification pattern & docs

## QA

- Pull code
- Run `gulp jekyll`
- Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/notifications/information/
- Run docs build
- Check that docs reflect correct notifications pattern 

## Details

Fixes #1060

### CODE REVIEW

**Comment by barrymcgee** [comment]
Demo: http://vanilla-framework-1060-notification-settings.demo.haus/vanilla-framework/examples/patterns/notifications/information/

**Comment by spencerbygraves** [comment]
The notification style shouldn't have an icon

**Comment by barrymcgee** [comment]
@spencerbygraves Removed icon; rebuilt demo.

File: docs/en/patterns/notification.md [hunk 1]
```diff
@@ -35,13 +35,22 @@ This negative variant should be used to convey information that is critical and
...
 <a href="https://vanilla-framework.github.io/vanilla-framework/examples/patterns/notifications/positive/"
     class="js-example">
     View example of the positive notification pattern
 </a>
 
+## .p-notification--information
+
+This information variant should be used to convey an information message.
+
+<a href="https://vanilla-framework.github.io/vanilla-framework/examples/patterns/notifications/positive/"
```
Comment (anthonydillon) [inline docs/en/patterns/notification.md]: This should be an information example.

**Comment by barrymcgee** [comment]
@anthonydillon Fixed..

**Pull request merged**